docs/source/building_an_experiment.rst

```diff
@@ -63,25 +63,25 @@ from highest level to lowest level:
 
    There are no requirements for slices interfaced in this manner.
 
-2. **INTTIME**
+2. **AVEPERIOD**
 
-   This type of interfacing allows for one slice to run its integration period (also known as integration time or averaging period), before switching to another slice's integration period. This type of interface effectively creates an interleaving scan where the scans for multiple slices are run 'at the same time', by interleaving the integration times.
+   This type of interfacing allows for one slice to run its averaging period (also known as integration time or integration period), before switching to another slice's integration period. This type of interface effectively creates an interleaving scan where the scans for multiple slices are run 'at the same time', by interleaving the averaging periods.
```

Comment:
```suggestion
   AVEPERIOD interfacing allows for one slice to run its averaging period (also known as integration time or integration period), before switching to another slice's averaging period. This type of interface effectively creates an interleaving scan where the scans for multiple slices are run 'at the same time', by interleaving the averaging periods.
```

Reply:
Above suggestion added capital letter reference to the interfacing type to follow the same format as subsequent types. Suggestion also fixes change mentioned in above comment.